src/components/Navabar.jsx

```diff
@@ -5,6 +5,10 @@ import '../../node_modules/jquery/dist/jquery.min.js';
 import '../../node_modules/bootstrap/dist/js/bootstrap.min.js';
 
  const Navbar = () => {
+  const clearLocalStorage = () => {
+    localStorage.clear()
```

Comment:
Instead of deleting the entire local storage, you could just remove the item you set

Reply:
This has been rectified